src/routes/workflow.js

```diff
@@ -147,6 +147,58 @@ function init (server, { middlewares, helpers } = {}) {
     const result = await requester.send(params)
     return result
   }))
+
+  // //////////// //
+  // WORKFLOW LOG //
+  // //////////// //
+
+  server.get({
+    name: 'workflow.listLogs',
+    path: '/workflow-logs'
+  }, checkPermissions([
+    'workflowLog:list:all'
+  ]), wrapAction(async (req, res) => {
+    const fields = [
+      'orderBy',
+      'order',
+      'page',
+      'nbResultsPerPage',
+
+      'id',
+      'createdDate',
+      'workflowId',
+      'eventId',
+      'runId',
+      'statusCode',
```

Comment:
I’d enable filtering by date rather than `statusCode` that can be missing (e.g. with skipped step).

Reply:
Ok thanks.